frontend/taipy-gui/base/src/app.ts

```diff
@@ -95,6 +96,15 @@ export class TaipyApp {
         }
         sendWsMessage(this.socket, "GMC", "get_module_context", { path: path }, this.clientId);
     }
+
+    trigger(actionName: string, triggerId: string, payload: Record<string, unknown> = {}) {
+        payload["action"] = actionName;
+        sendWsMessage(this.socket, "A", triggerId, payload, this.clientId, this.context);
+    }
+
+    uploadFile(encodedName: string, files: FileList, progressCallback: (val: number) => void) {
```

Comment:
Rename to 'upload()'? I don't see the need for 'File'

Reply:
I will rename for this one! tx Fabien